From c253a87d9ac06ecb2c58df894b8e971b6b543bcd Mon Sep 17 00:00:00 2001
From: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Date: Wed, 7 Feb 2018 23:33:27 +0700
Subject: [PATCH 714/909] arm64: dts: r8a77990: Support IPMMU(MMU mode) for
 IPMMU-VP0 and IPMMU-VC0

commit e31145efd466e4c78d9de5853de60d95fb6ad43a from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch enables MMNGR support for IPMMU(MMU mode) in IPMMU-VP0 and
IPMMU-VC0.

Signed-off-by: Hai Nguyen Pham <hai.pham.ud@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990.dtsi b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
index 99c846ea3de0..bd4d71e2b93e 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77990.dtsi
@@ -847,7 +847,7 @@
 		};
 
 		ipmmu_vc0: mmu@fe6b0000 {
-			compatible = "renesas,ipmmu-r8a77990";
+			compatible = "renesas,ipmmu-mmu-r8a77990";
 			reg = <0 0xfe6b0000 0 0x1000>;
 			renesas,ipmmu-main = <&ipmmu_mm 12>;
 			power-domains = <&sysc R8A77990_PD_A3VC>;
@@ -865,7 +865,7 @@
 		};
 
 		ipmmu_vp0: mmu@fe990000 {
-			compatible = "renesas,ipmmu-r8a77990";
+			compatible = "renesas,ipmmu-mmu-r8a77990";
 			reg = <0 0xfe990000 0 0x1000>;
 			renesas,ipmmu-main = <&ipmmu_mm 16>;
 			power-domains = <&sysc R8A77990_PD_ALWAYS_ON>;
-- 
2.17.1

